dev/build/windows/makecoq_mingw.sh

```diff
@@ -1281,7 +1285,6 @@ function get_cygwin_mingw_sources {
 
 function make_coq_installer {
   make_coq
-  make_mingw_make
```

Comment:
I'm fine with that, but IIRC you did want this line to stay...

Reply:
I found that it is redundant. There is this code in makecoq_mingw.sh

if [ "$INSTALLMAKE" == "Y" ] ; then
  make_mingw_make
fi

before the call to make_coq_installer. So if INSTALLMAKE is Y, make will be built, otherwise not.

Sorry that I didn't review your changes properly last time!

Reply:
I did not spot it either, no problem. I'll merge as soon as CI says yes.